Accessibility and html pages in docs (#611)
arXiv/arxiv-docs: source/about/accessible_HTML.md

# HTML as an accessible format for papers

The message we have heard through our accessibility research and from our community is that arXiv can have the most impact by offering HTML alongside the existing PDF and source files. This is the project arXiv is actively engaged in now. Accessible research papers is a goal within reach!
{ .intro }

## Latest Updates
The HTML papers project is in Closed Beta stage, soon to be Public Beta. We will be making links to HTML papers available on the live abstract pages, the list page, and as a part of the submission process. We know there will be bugs, so why not wait until it is perfect?
1. First, because our community asked us not to. They told us: "don't let perfect be the enemy of good. We need HTML now."
2. And second, because we need your help identifying rendering issues. The low hanging fruit is done. We are now in the realm of identifying specific LaTeX packages that have issues and relaying that information to the NIST team behind LaTeXML, the conversion tool from LaTeX to HTML.

> From this vantage point, we are asking authors to tolerate the bugs in the HTML renderings of their papers. Often times, the HTML will not perfectly match the pdf in many cases, but we will actively advise readers that the feature is in beta. A header message and beta banner will provide context to these flaws such that they do not reflect poorly on authors, and instead promote these authors' proclivity for open science.

Author: ``
```suggestion
We ask authors to focus on content and accuracy of data, not on the visual display of the HTML. Our goal is not to match the PDF display, but to offer a useful format that expands access to people with disabilities who rely on assistive technology like screen readers, and to readers on mobile devices. If you notice that a formula or other content rendered incorrectly, your feedback will help us improve this service.
```

## How to provide feedback
We invite you to provide feedback on the accessibility and accuracy of the HTML by following these simple steps:
1. Go to the abstract page for a paper you are interested in reading.
2. Look in the section where you find the link to the PDF download, and below that you will see a new link to the HTML version.

> 3. The HTML page will load, and look for the button to report issues, or click [KEYBOARD COMMAND] to pull up an accessible and keyboard-friendly reporting method. You can also click [OTHER KEYBOARD COMMAND] anytime while on an HTML page to pull up a glossary of keyboard commands for reporting.

Author: ```suggestion
3. On the HTML page, find the button to report issues, or click [KEYBOARD COMMAND] to pull up an accessible and keyboard-friendly reporting method. You can also find a quick glossary of keyboard commands in the HTML page footer.
```
